test/blas.jl

```diff
@@ -3,6 +3,8 @@ using EnzymeTestUtils
 using LinearAlgebra
 using Test
 
+# LLVM ERROR: out of memory LLVMWriteBitcodeToMemoryBuffer
+if sizeof(Int) != sizeof(Int64)
```

Comment:
Alternatively
```suggestion
if Sys.WORD_SIZE != 64
```
This is very similar to the test you wrote ([`Sys.WORD_SIZE` is based on the size of `Int`](https://github.com/JuliaLang/julia/blob/7fa26ce89dc09596e865123afe37fe5a5cd4bd9a/base/sysinfo.jl#L105)), but I think the intent is more explicit